wordpress-mobile/WordPress-iOS
# Activity Log feed endpoint remote implementation

First step in #7832 
Remote implementation of the activity log feed endpoint.

**To test:**
You can add a call to `ActivityServiceRemote.getActivityForSite` and execute it for a site that supports activity log, or you can just run the tests :)
Check that other calls to the REST API continue to work correctly since the mechanism to create the URLs was slightly modified.

Needs review: @koke
